KomaMRIPlots/src/ui/DisplayFunctions.jl

```diff
@@ -1080,38 +1080,42 @@ function plot_phantom_map(
         unit = " ms"
         if key == :T1
             cmax_key = 2500 / factor
-            colors = MAT.matread(path * "/assets/T1cm.mat")["T1colormap"][1:70:end, :]
-            N, _ = size(colors)
-            idx = range(0, 1; length=N) #range(0,T,N) works in Julia 1.7
+            colors = relaxationColorMap("T1")
+            N = length(colors)
+            idx = range(0, 1; length=N)
+        
+            # Create the colormap
             colormap = [
                 (
                     idx[n],
-                    string("rgb(", 
-                        floor(Int, colors[n,1] * 255), ",",
-                        floor(Int, colors[n,2] * 255), ",",
-                        floor(Int, colors[n,3] * 255), ")"
-                        )
-                ) 
+                    string("rgb(",
+                        floor(Int, colors[n].r * 255), ",",
+                        floor(Int, colors[n].g * 255), ",",
+                        floor(Int, colors[n].b * 255), ")"
+                    )
+                )
```

Comment:
Can't you just use the color list? Same for T2.

Reply:
No, it can't be done.

Reply:
It can't be done, but I simplified the use of the color list in commit [479f317](https://github.com/JuliaHealth/KomaMRI.jl/pull/528/commits/479f31791aa5cec0e9af61a9e83d1bf99f680cd2)